### PR TITLE
[sigh] Migrated binary to use commands_generator

### PR DESCRIPTION
**Pull request by KrauseFx**
Related to https://github.com/fastlane/fastlane/pull/4514

### CODE REVIEW

File: sigh/lib/sigh.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 require 'sigh/resign'
 require 'sigh/manager'
 require 'sigh/options'
+require 'sigh/local_manage'
```
Comment (mfurtak) [inline sigh/lib/sigh.rb]: What is this?

Comment (KrauseFx) [inline sigh/lib/sigh.rb]: Up until now this was imported on https://github.com/fastlane/fastlane/pull/4579/files#diff-1da9a235513ffba6a58c0e444473ad29L34. Instead it's much cleaner to import it in `sigh.rb` with all the other imports.

**Comment by mfurtak** [comment]
Gotcha thanks! 👍  :ship: it

**Comment by KrauseFx** [comment]
Thanks 👍

**Pull request merged**